R/utils_insert_db.R

```diff
@@ -183,22 +183,32 @@ insert_riskmetric_to_db <- function(pkg_name,
     # If the metric is not part of the assessment, then skip iteration.
     if(!(metric$name %in% colnames(riskmetric_score))) next
     
+    # attempt to get export_help data from pkg_install
+    exp_help <- function(pkg_name) {
+      if (is.na(riskmetric_assess$export_help[[1]][1])) {
+      valu <- riskmetric::assess_export_help(riskmetric::pkg_ref(pkg_name)) %>% 
+        riskmetric::metric_score()
+      as.character(round(valu*100, 2))
+      } else {
+        return(riskmetric_assess$export_help[[1]][1])
+      }
+    }
+
```

Comment:
The idea here was to populate export_help -- in some cases it was returning NA

Reply:
Yes, I think it's a deceiving to show this metric if `riskmetric` isn't including it in the risk score. Unless you could replace that section of the pkg_ref object before it's assessed and scored, right?

Also, why does `riskmetric::assess_export_help(riskmetric::pkg_ref(pkg_name))` work in this case? Does it only work because the pkg is installed on our local machines? This wouldn't work on shinyapps.io, correct?

Reply:
Without this, I end up showing NA on the maintenance metrics panel instead of a real value.

For example
![image](https://github.com/pharmaR/riskassessment/assets/75215958/0a8011a6-fdfc-44b9-a6e1-477bcff11756)

Reply:
exp_help function removed